src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+// Select the logo element
+const logo = document.querySelector('.logo');
+
+const clickPromise = new Promise((resolve, reject) => {
```

Comment:
It's not recommended to create promises that can potentially never be resolved or rejected. In this case, if the user never clicks the logo, the promise will be pending indefinitely. It's better to ensure that a promise always settles, to avoid potential memory leaks and unpredictable behavior.